ortools/sat/python/cp_model.py

```diff
@@ -654,6 +654,10 @@ def __init__(self, model, start_index, size_index, end_index,
         if name:
             self.__ct.name = name
 
+        self.__start = start
```

Comment:
Sorry, I have to disagree with this change. As you can change the proto behind the back, I would prefer recreating the IntVar on demand.

Reply:
True, I didn't think about a possible mismatch and this is quite easy to do outside the library anyway.